lib/toy_minefield.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'minefield'
+
```

Comment:
So was toy minefield a smaller minefield for you to play around with in user testing? This could be awesome if you wanted to manually check victory conditions.